httpdocs/js/webots-cloud.js

```diff
@@ -1250,6 +1486,304 @@ ${deleteProject}`;
+          // remove intial '*field' string
+          fieldString = fieldString.replace(removeInitialFieldRegex, '  ');
+          fieldString = fieldString.replace('webots://', 'https://raw.githubusercontent.com/cyberbotics/webots/released');
+
+          // remove unwanted spaces between field type and field name (if needed)
+          if (spacesToRemove > 0)
+            fieldString = fieldString.replace(fieldType + ' '.repeat(spacesToRemove), fieldType);
+
+          fields += fieldString + '\n';
+        }
+      }
+      fetch('ajax/proto/documentation.php', {method: 'post', body: JSON.stringify({url: protoURl})})
+        .then(function(response) {
+          return response.json();
+        })
+        .then(content => {
+          const baseType = content.base_type;
+          const description = content.description;
+          file += description + '\n\n';
+          file += 'Derived from [' + baseType + '](https://cyberbotics.com/doc/reference/' + baseType?.toLowerCase() + ').\n\n';
+          file += '```\n';
+          file += protoName + ' {\n';
+          file += fields;
+          file += '}\n';
+          file += '```\n\n';
+
+          if (describedField) {
+            file += '### ' + protoName + ' Field Summary\n\n';
+            for (const [fieldType, fieldName, fielDescription] of describedField) {
+              file += '- `' + fieldName + '` : ' + fielDescription;
+              const isMFField = fieldType.startsWith('MF');
+              if (fieldEnumeration.has(fieldName)) {
+                const values = fieldEnumeration.get(fieldName);
+                if (isMFField)
+                  file += ' This field accept a list of ';
+                else {
+                  if (values.length > 1)
+                    file += ' This field accepts the following values: ';
+                  else
+                    file += ' This field accepts the following value: ';
+                }
+
+                for (let i = 0; i < values.length; i++) {
+                  const value = values[i].split('{')[0]; // In case of node keep only the type
+                  if (i === values.length - 1) {
+                    if (isMFField)
+                      file += '`' + value.trim() + '` ' + fieldType.replace('MF', '').toLowerCase() + 's.';
+                    else
+                      file += '`' + value.trim() + '`.';
+                  } else if (i === values.length - 2) {
+                    if (values.length === 2)
+                      file += '`' + value.trim() + '` and ';
+                    else
+                      file += '`' + value.trim() + '`, and ';
+                  } else
+                    file += '`' + value.trim() + '`, ';
+                }
+              }
+              file += '\n\n';
+            }
+          }
+
+          const license = content.license;
+          const licenseUrl = content.license_url;
+          let version = content.version;
```

Comment:
Why can't we make it const?
```suggestion
          const version = content.version;
```